logging/logging.go

```diff
@@ -304,6 +306,8 @@ func initLogger() (err error) {
 
 // GetLogger returns the current logger instance
 func GetLogger() (log *Logger) {
+	lock.RLock()
+	defer lock.Unlock()
```

Comment:
There is an issue here RLock/Unlock. I'm not sure we should add a mutex here as we never saw this issue before, so I guess there something to do in the test suite regarding the logger initialization.

Reply:
@safchain - removed